bin/tictactoe

```diff
@@ -1,3 +1,10 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+again = "Y"
+while again == "Y"  
+  Game.new.start
+  puts "Would you like to play again? (Y/N)"
+  again = gets.strip.upcase
```

Comment:
Prefer to encapsulate CLI logic into a class. For example the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in call.